warehouse/utils/__init__.py

```diff
@@ -14,7 +14,8 @@
 
 
 def now() -> datetime.datetime:
-    return datetime.datetime.now(datetime.UTC)
+    """Return the current datetime in UTC without a timezone."""
+    return datetime.datetime.now(datetime.UTC).replace(tzinfo=None)
```

Comment:
This is now just the same as `datetime.datetime.now()`, unless I'm missing something.

Reply:
Does this help explain the difference?
```python
>>> import datetime
>>> print(datetime.datetime.now())
2023-11-06 17:09:32.924917
>>> print(datetime.datetime.now(datetime.UTC).replace(tzinfo=None))
2023-11-06 22:09:45.453366
```

Reply:
(and yes, [time is hard](https://zachholman.com/talk/utc-is-enough-for-everyone-right))